cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -90,6 +99,21 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.depends("variable_id", "option_id")
+    def _compute_value_char(self):
+        for rec in self:
+            if rec.variable_id.option_ids and rec.option_id:
```

Comment:
-> You check `variable_id.option_ids` here, which means that if it changes you should recompute the field value too.